aapcs64-morello/aapcs64-morello.rst

```diff
@@ -808,7 +815,7 @@ The ``va_list`` type may refer to any parameter in a parameter list. All Anonymo
 The va_start() macro
 --------------------
 
-The ``va_start`` macro shall initialize the ``va_list`` argument to the value of C9 as seen in the entry of the callee.
+The ``va_start`` macro shall initialize the ``va_list`` argument to the value of derived from C9 with the address of the first Anonymous argument.
```

Comment:
s/of derived/derived/
It also feels strange to say *the* value derived from [...], rather *a* value? Using "value" feels a bit confusing too as it seems to refer to the capability address in other places.